umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -6,11 +6,60 @@
   [cmr.umm-spec.date-util :as date-util]
   [cmr.umm-spec.umm-spec-core :as spec-core]))
 
+(def DataCenterURL
+  "Data center URL's content type."
+  "DataCenterURL")
+
+(def HomePage
+  "Data center URL's type."
+  "HOME PAGE")
+
 (defn field-update-functions
   "Partial update functions for handling specific update cases"
   [umm]
   {[:Instruments] (partial util/update-in-each umm [:Platforms])})
 
+(defn- get-home-page-url
+  "Get data center home page url from a list of related urls."
+  [related-urls]
+  (when (seq related-urls)
+    (first (for [{:keys [URLContentType Type URL]} related-urls
+                 :when (and (= DataCenterURL URLContentType)
+                            (= HomePage Type))]
+             {:URLContentType DataCenterURL
+              :Type HomePage
+              :URL URL})))) 
+
+(defn- home-page-url?
+  "Check if a related url is a data center home page url."
+  [related-url]
+  (let [{:keys [URLContentType Type URL]} related-url]
+    (if (and (= DataCenterURL URLContentType)
```

Comment:
I've seen this logic a few times in the CMR where we do an `if` then `true` or `false`. I would just remove all that and return:

`(and (= DataCenterURL URLContentType)
          (= HomePage Type))`

Anyone have a strong style preference?

Reply:
I could go either way. The if then one is more intuitive, this one is more compact. I will change it. Done.